src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -438,8 +438,42 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
                 null;
             $product['product_quantity_refunded'] = $product['product_quantity_refunded'] ?: false;
 
+            $productType = !empty($product['pack_items']) ? OrderProductForViewing::TYPE_PACK :
+                OrderProductForViewing::TYPE_PRODUCT_WITHOUT_COMBINATIONS;
+
             $orderInvoice = new OrderInvoice($product['id_order_invoice']);
 
+            $packItems = [];
+            foreach ($product['pack_items'] as $pack_item) {
+                $packItemType = !empty($pack_item['pack_items']) ? OrderProductForViewing::TYPE_PACK :
+                    OrderProductForViewing::TYPE_PRODUCT_WITHOUT_COMBINATIONS;
+                $packItemImagePath = isset($pack_item['image_tag']) ?
+                    $this->imageTagSourceParser->parse($pack_item['image_tag']) :
+                    null;
+                $packItems[] = new OrderProductForViewing(
+                    null,
+                    $pack_item['id_product'],
+                    $pack_item['name'],
+                    $pack_item['reference'],
+                    $pack_item['supplier_reference'],
+                    $pack_item['pack_quantity'],
+                    0,
+                    0,
+                    $pack_item['current_stock'],
+                    $packItemImagePath,
+                    Tools::ps_round(0, 2),
```

Comment:
Shouldn't we be using `Context::getContext()->getComputingPrecision()` for the rounding precision here?

Reply:
Done. Thanks